src/commands/data/connectors/update.ts

```diff
@@ -9,10 +9,10 @@ interface Dict<T> {
 }
 
 export default class ConnectorsUpdate extends BaseCommand {
-  static description = 'Update the settings for a Data Connector\nRead more about this feature at https://devcenter.heroku.com/articles/heroku-data-connectors#update-configuration'
+  static description = 'Update the settings, tables and columns to exclude for a Data Connector\nRead more about this feature at https://devcenter.heroku.com/articles/heroku-data-connectors#update-configuration'
```

Comment:
We should make sure to update the devcenter article when this ships.

Reply:
Yup, I will write a draft now. But can't publish that till this PR is added.